app/views/layouts/_header.html.erb

```diff
@@ -1775,7 +1775,7 @@
           <%= link_to t(".profile_account"), edit_user_path(current_user),  class: "dropdown-item"%>
           <hr class="dropdown-divider" />
           <%= link_to t(".Sign_out"), logout_path,
```

Comment:
Đoạn này không phải code c, nhưng tên biến yml không được đặt in hoa thì phải ạ

Reply:
done nha